conf/nickserv.example.json

```diff
@@ -1,3 +1,5 @@
 {
-	"password" : "dicks"
+  "nickserv": "nickserv",
+  "password" : "dicks",
+  "successtext": "you are now identified for "
```

Comment:
perhaps this should be a regexp? still a string in the config, but always passed to `new RegExp` and used with `.test`

Reply:
Works for me.

Reply:
Done.